ros2-library/src/main/java/us/ihmc/ros2/QueuedROS2Publisher.java

```diff
@@ -49,11 +49,10 @@ public boolean publish(T data)
          }
          else
          {
-            if (throttleStopwatch.totalElapsed() > 1.0)
+            if (errorPrintThrottler.run(0.5))
```

Comment:
1 second not fast enough? Honestly, I'd almost agree to add a 5 print limit

Reply:
It's 0.5 hz

Reply:
I'd double check that. I thought run method took in the period

Reply:
Ah yes, Dex tricked me, that is the period